### PR TITLE
Zeromq+protobuf/json publishing added as an option

### PR DESCRIPTION
**Pull request by kobzev**

### CODE REVIEW

File: exchangeEventsHandler.js [hunk 1]
```diff
@@ -144,15 +149,32 @@ class ExchangeEventsHandler {
...
             if (!this._settings.SocketIO.Disabled && this._socketio != null)
                 this._socketio.sockets.send(orderBook);
 
+            if (!this._settings.ZeroMq.Disabled && this._zeroMq != null) {
+                if (this._settings.ZeroMq.Serializer == "protobuf") {
+                    const protoOrderBook = this._mapPublishOrderBookToProtobufOrderBook(orderBook, timestamp)
+                    var payload = this._orderbookResponse.create({order_books: [protoOrderBook]})
+                    const message = this._orderbookResponse.encode(payload).finish();
+                    this._zeroMq.send(["orderbooks", message]);
+                    // this._log.debug(`Order Book: ${protoOrderBook.source} ${protoOrderBook.asset}, bids:${protoOrderBook.bids.length}, asks:${protoOrderBook.asks.length}, best bid:${protoOrderBook.bids[0].price}, best ask:${protoOrderBook.asks[0].price}, timestamp: ${JSON.stringify(protoOrderBook.timestamp)}.`)
```
Comment (AlexeyZelenin) [inline exchangeEventsHandler.js:170]: Uncomment please

File: exchangeEventsHandler.js [hunk 2]
```diff
@@ -144,15 +149,32 @@ class ExchangeEventsHandler {
...
+                }
+                else if (this._settings.ZeroMq.Serializer == "json") {
+                    this._zeroMq.send(["orderbooks", JSON.stringify(orderBook)]);
+                    // this._log.debug(`Order Book: ${orderBook.source} ${orderBook.asset}, bids:${orderBook.bids.length}, asks:${orderBook.asks.length}, best bid:${orderBook.bids[0].price}, best ask:${orderBook.asks[0].price}, timestamp: ${orderBook.timestamp}.`)
```
Comment (AlexeyZelenin) [inline exchangeEventsHandler.js:174]: Uncomment please

File: gRPC/orderbooks.proto [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+}
+
+message OrderBook {
+    string source = 1;
+    AssetPair assetPair = 2;
+    repeated LimitOrder bids = 3;
+    repeated LimitOrder asks = 4;
+    google.protobuf.Timestamp timestamp = 5;
+    google.protobuf.Timestamp timestampMs = 6;
+    google.protobuf.Timestamp timestampin = 7;
```
Comment (AlexeyZelenin) [inline gRPC/orderbooks.proto:25]: Please change to timestampIn

File: zeromq/zeromq.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+  const isDisabled = settings.ZeroMq.Disabled
+  const port = settings.ZeroMq.Port
+  
+  zeromq = zmq.socket("pub");
+  if (!isDisabled){
+    zeromq.bindSync("tcp://127.0.0.1:" + port);
+  }
+
+  _log.info(`Publisher bound to port ${port}`)
```
Comment (AlexeyZelenin) [inline zeromq/zeromq.js:23]: Please change message to:
ZeroMQ listening at 'tcp://127.0.0.1:${port}'

File: exchangeEventsHandler.js (hunk 1 above)
Comment (kobzev) [reply exchangeEventsHandler.js]: done

File: exchangeEventsHandler.js (hunk 2 above)
Comment (kobzev) [reply exchangeEventsHandler.js]: done

File: gRPC/orderbooks.proto (hunk 3 above)
Comment (kobzev) [reply gRPC/orderbooks.proto]: done on both ends

File: zeromq/zeromq.js (hunk 4 above)
Comment (kobzev) [reply zeromq/zeromq.js]: Changed to 
`ZeroMQ is publishing from 'tcp://127.0.0.1:${port}`

**Pull request merged**